packages/stryker/test/helpers/producers.ts

```diff
@@ -236,5 +236,14 @@ export const testableMutant = (fileName = 'file') => new TestableMutant('1337',
   textFile({ name: fileName, content: 'const a = 4 + 5' })
 ));
 
+export const excludedTestableMutant = (fileName = 'file') => new TestableMutant('2448', mutant({
```

Comment:
I would rather add the parameter `mutatorName` to the `testableMutant` factory method. That way, it is really clear from the test code what is happening.